src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
+
+    :param obj: The object to be filtered
+    :return: The object with keys filtered out
+    """
+    return {k: v for k, v in obj.items() if not k.startswith("__")}
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu.
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :return: Tuple int used to color the menu
+    """
+    if entry["is_default"]:
+        return 2, 0
+    return 3, 0
+
+
+def content_heading(obj: Any, screen_w: int) -> CursesLines:
+    """Create a heading for the setting entry showing.
+
+    :param obj: The content going to be shown
+    :param screen_w: The current screen width
+    :return: The heading
+    """
+    heading = []
+    string = obj["name"].replace("_", " ")
+    if obj["is_default"]:
+        string += f" (current/default: {obj['current_value']})"
+        color = 2
+    else:
+        string += f" (current: {obj['current_value']})  (default: {obj['default']})"
+        color = 3
+
+    string = string + (" " * (screen_w - len(string) + 1))
+
+    heading.append(
+        tuple(
+            [
+                CursesLinePart(
+                    column=0,
+                    string=string,
+                    color=color,
+                    decoration=curses.A_UNDERLINE,
+                ),
+            ],
+        ),
```

Comment:
This nested thing can be simplified by declaring a tuple literal directly instead of using a `tuple()` constructor with a temporary "throw-away" in-memory list:
```suggestion
        (
            CursesLinePart(
                column=0,
                string=string,
                color=color,
                decoration=curses.A_UNDERLINE,
            ),
        ),
```

Reply:
Good catch. I changed this to just a tuple literal declaration.